pkg/provisioner/ironic/ironic.go

```diff
@@ -1919,6 +1919,16 @@ func (p *ironicProvisioner) IsReady() (result bool, err error) {
 }
 
 func (p *ironicProvisioner) HasCapacity() (result bool, err error) {
+	bmcAccess, err := p.bmcAccess()
+	if err != nil {
+		return false, err // shouldn't actually happen so late in the process
+	}
+
+	// If the current host uses virtual media, do not limit it. Virtual
+	// media deployments may work without DHCP and can share the same image.
+	if bmcAccess.SupportsISOPreprovisioningImage() {
```

Comment:
General question: the delay mechanism gets applied also during deprovisioning, did you maybe also evaluate the impact of disabling it for vmedia hosts also during this phase?

Reply:
It matters even less there: cleaning works completely locally.